readme.md

```diff
@@ -64,7 +66,7 @@ Your Answer:
 Describe the purpose of views and models in FE JS.
 
 Your Answer:
-```text
+```Front End JavaScript models are the structure that define a given data set. Front End JS views get closer to structuring the DOM's actual layout of the selected data models.
 
```

Comment:
Pretty much. Views not only render models but also define functionality triggered by user interaction (e.g., event listeners).